src/pages/my_page/user_profile/MyResults.tsx

```diff
@@ -32,6 +34,15 @@ export const MyResults = () => {
     rowsPerPage * (page - 1)
   );
 
+  const promotedPublicationsQuery = useQuery({
+    queryKey: ['person-preferences', personId],
+    queryFn: () => fetchPromotedPublicationsById(personId),
+    meta: { errorMessage: false },
+    retry: false,
+  });
+
+  const promotedPublications = promotedPublicationsQuery.data?.promotedPublications ?? [];
```

Comment:
her settes `promotedPublications` til tomt array om det er `undefined`. Fjerner ikke det behovet for å kunne ta i mot `undefined` i `RegistrationSearchResults`? Eller brukes den andre plasser hvor det trengs kanskje?

Reply:
for å snu spørsmålet litt, skulle propen egentlig vært `undefined` her, eller er det riktig med tomt array? (har ikke fått tid til å satt meg godt nok inn i denne enda, kan godt være at tomt array er det vi ønsker for en grunn jeg ikke vet)